### PR TITLE
Create count AP function

### PR DESCRIPTION
**Pull request by eurekaneuro**

### CODE REVIEW

**Comment by neurodroid** [comment]
In your pull request, can you directly edit `stimfit/src/stimfit/py/spells.py` so that we see a diff rather than a new code block? Otherwise it's difficult to tell what you changed and what the issue is.

File: count AP function [hunk 1]
```diff
@@ -91,3 +93,75 @@ def counting():
...
+
+    pstart = int(round(start/dt))
+    pdelta = int(round(delta/dt))
+
+    # select the section of interest within the trace
+    selection = stf.get_trace()[pstart:(pstart+pdelta)]
+
+    # algorithm to detect events
+    event_counter, i = 0, 0 # set counter and index to zero
+
```
Comment (eurekaneuro) [inline count AP function]: added code
event_time = ""
return_value = ""

**Comment by neurodroid** [comment]
In your fork (`patch-1`), please edit `stimfit/src/stimfit/py/spells.py` directly so that we can get a proper diff. Or alternatively, send me the result of
`diff -u stimfit/src/stimfit/py/spells.py /path/to/your/version/of/spells.py`
Without a proper diff, it's impossible to say why you're not seeing the results that you expect.

**Comment by neurodroid** [comment]
Closing as there has been no update in more than half a year.

**Pull request closed without merging**